<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>228</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>291</width>
      <height>229</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_6">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid gray;
}

QPushButton: pressed {
	background-color: qlineneargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_percent">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	color: rgb(0, 0, 0);
	border: 1px solid gray;
}

QPushButton: pressed {
	background-color: qlineneargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
       </property>
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QPushButton" name="pushButton_0">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid gray;
}

QPushButton: pressed {
	background-color: qlineneargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="pushButton_plus">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgb(215, 151, 57);
	color: white
	border: 1px solid gray;
}

QPushButton: pressed {
	background-color: qlineneargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButton_1">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid gray;
}

QPushButton: pressed {
	background-color: qlineneargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="pushButton_equal">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgb(215, 151, 57);
	color: white
	border: 1px solid gray;
}

QPushButton: pressed {
	background-color: qlineneargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_7">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid gray;
}

QPushButton: pressed {
	background-color: qlineneargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_8">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid gray;
}

QPushButton: pressed {
	background-color: qlineneargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_ac">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	color: rgb(0, 0, 0);
	border: 1px solid gray;
}

QPushButton: pressed {
	background-color: qlineneargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
       </property>
       <property name="text">
        <string>A/C</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_9">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid gray;
}

QPushButton: pressed {
	background-color: qlineneargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="pushButton_minus">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgb(215, 151, 57);
	color: white
	border: 1px solid gray;
}

QPushButton: pressed {
	background-color: qlineneargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="pushButton_2">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid gray;
}

QPushButton: pressed {
	background-color: qlineneargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="pushButton_3">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid gray;
}

QPushButton: pressed {
	background-color: qlineneargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_5">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid gray;
}

QPushButton: pressed {
	background-color: qlineneargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="pushButton_multiply">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgb(215, 151, 57);
	color: white
	border: 1px solid gray;
}

QPushButton: pressed {
	background-color: qlineneargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_plus_minus">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	color: rgb(0, 0, 0);
	border: 1px solid gray;
}

QPushButton: pressed {
	background-color: qlineneargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
       </property>
       <property name="text">
        <string>+/-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_4">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid gray;
}

QPushButton: pressed {
	background-color: qlineneargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="pushButton_divide">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgb(215, 151, 57);
	color: white
	border: 1px solid gray;
}

QPushButton: pressed {
	background-color: qlineneargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FF7832, stop: 1 #FF9739);
}</string>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="pushButton_dot">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgb(215, 215, 215);
	color: rgb(0, 0, 0);
	border: 1px solid gray;
}

QPushButton: pressed {
	background-color: qlineneargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="4">
      <widget class="QLabel" name="result_show">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border-bottom: 1px solid gray;
}

background-color: white;</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="margin">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
